docs/en/user_guides/inference.md

```diff
@@ -0,0 +1,148 @@
+# Inference
+
+Currently, we support various popular generative models, including unconditional GANs, conditional GANs, image translation models, internal GANs and diffusion models.
+In this section, we will specify how to sample fake images by using our existing models. For model inference, all of the APIs have been included in [mmgen/apis/inference.py](https://github.com/open-mmlab/mmgeneration/tree/1.x/mmgen/apis/inference.py). The most important function is `init_model` for creating a generative model from a config. Then, adopting the sampling function in this file with the generative model will offer you the synthesized images.
```

Comment:
https://github.com/open-mmlab/mmgeneration/tree/1.x/mmgen/apis/inference.py ->
Can we use a relative link for this url?